docs/iris/src/whatsnew/1.9.rst

```diff
@@ -0,0 +1,13 @@
+Iris 1.9 Bugs Fixed
+===================
+Iris 1.9 Features
```

Comment:
The order in the 'List of Dicts' at the beginning of the code, defines the order of the sections - so they can be kept there. They're basically random at the moment.

I hadn't thought about controlling the order of sections through the snippet structure, but the first thought that comes to mind is that it's going to make things very inhomogenous between different releases' Whats New pages. Specifying a priority into the snippet names would also be feasible, and would allow the removal/simplification of the regex discussed below - if this is preferred to 'Date order'.